test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs

```diff
@@ -109,7 +109,7 @@ public void TestWillCreateTrxLoggerInTheSpecifiedResultsDirectoryBySwitch()
             // Call test with trx logger enabled and results directory explicitly specified.
             CommandResult result = new DotnetTestCommand()
                                        .WithWorkingDirectory(testProjectDirectory)
-                                       .ExecuteWithCapturedOutput("--logger:trx -r \"" + trxLoggerDirectory + "\"");
+                                       .ExecuteWithCapturedOutput("--logger trx -r \"" + trxLoggerDirectory + "\"");
```

Comment:
This was a conflict from https://github.com/dotnet/cli/commit/cb52224272cda0e148dbd0f88332f21dcc508750#diff-56c1a6ebe6851ff03faea7b182cb783e.